pandas/io/formats/style.py

```diff
@@ -576,10 +576,10 @@ def apply(self, func, axis=0, subset=None, **kwargs):
             on ``axis``), and return an object with the same shape.
             Must return a DataFrame with identical index and
             column labels when ``axis=None``
-        axis : int, str or None
-            apply to each column (``axis=0`` or ``'index'``)
-            or to each row (``axis=1`` or ``'columns'``) or
-            to the entire DataFrame at once with ``axis=None``
+        axis : {0 or ‘index’, 1 or ‘columns’, None}, default 0
+            apply to each column (``axis=0`` or ``'index'``), to each row
```

Comment:
I'm actually surprised this isn't failing CI due to the description not starting with a capital letter. Not something to fix here but I wonder if there's something awry with CI @datapythonista

Reply:
Scratch that looks this PR is still outstanding (#25232)